test/langtools/jdk/javadoc/lib/javadoc/tester/LinkChecker.java

```diff
@@ -351,7 +351,7 @@ void addID(int line, String name) {
 
         void addReference(String name, Path from, int line) {
             if (checked) {
-                if (name != null && !name.isEmpty()) {
+                if (name != null) {
```

Comment:
Good to see the checks are back! (https://github.com/openjdk/jdk/pull/6011#discussion_r732039878)